{
    "id": 130087,
    "name": "CVE-2019-16729",
    "cwe": "NVD-CWE-noinfo",
    "modified": "2023-02-27T15:30:35",
    "published": "2019-09-24T03:15:11",
    "status": "Analyzed",
    "summary": "pam-python before 1.0.7-1 has an issue in regard to the default environment variable handling of Python, which could allow for local root escalation in certain PAM setups.",
    "cvss3": {
        "cvss": 7.8,
        "exploitability_score": 1.8,
        "impact_score": 5.9,
        "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
    },
    "cvss2": {
        "exploitability_score": 3.9,
        "impact_score": 10.0,
        "cvss": 7.2,
        "vector": "AV:L/AC:L/Au:N/C:C/I:C/A:C"
    },
    "cpes": [
        {
            "id": 109409
        },
        {
            "id": 124875
        },
        {
            "id": 193709
        },
        {
            "id": 193717
        },
        {
            "id": 386808
        },
        {
            "id": 386809
        },
        {
            "id": 386810
        },
        {
            "id": 386811
        },
        {
            "id": 587080
        }
    ]
}